schemas/validation_tests/individual-valid-1.json

```diff
@@ -4,6 +4,12 @@
         "center_name" : "EBI-TEST",
         "ega_accession": "EGAI00000000001"
     },
+    "schema_descriptor": {
+        "object_type": "individual",
+        "described_by_schema_uri": "https://github.com/EbiEga/ega-metadata-schema/tree/main/schemas/EGA.individual.json",
+        "object_schema_version": "0.0.0",
+        "common_schema_version": "0.0.0"
+    },
```

Comment:
I know the order of the attributes doesn't matter, but it might be nice to put this particular block of "business metadata" at the end of the examples. That way the metadata that is actually about the object and likely what the user is interested in readying is all together at the top of the document.

Reply:
On it 👍